### ISSUE TITLE
imgproxy health doesn't work when IMGPROXY_PATH_PREFIX is defined

### ISSUE THREAD

**Issue by shadowlmd** [imgproxy/imgproxy#698]
How to test:
```
shadowlmd@home:~$ docker run --rm -e IMGPROXY_PATH_PREFIX=/i -d darthsim/imgproxy
ac067a0921d489332ece71caca58f09279907af3bafbdfbf0f99c5fcc5bb2f6e
```
```
shadowlmd@home:~$ docker container exec -it objective_dubinsky bash
root@ac067a0921d4:/# imgproxy health

root@ac067a0921d4:/# imgproxy health || echo fail

fail
```
The endpoint is available at /i/health in this case and can be queried using curl for example.

**Comment by DarthSim** [comment]
Sorry, I forgot to mention that this was fixed in `v3.0.0`

**Comment by shadowlmd** [comment]
Thanks, it works now. :)
